packages/wonder-blocks-tooltip/components/tooltip-popper.js

```diff
@@ -7,23 +7,21 @@ import * as React from "react";
 import {Popper} from "react-popper";
 import type {PopperChildrenProps} from "react-popper";
 
-import TooltipBubble from "./tooltip-bubble.js";
 // NOTE(jeff): Here we share some code for use with PopperJS. Long term,
 // we should either contribute this code to the PopperJS component, or its
 // own non-wonder-blocks package.
 // $FlowIgnoreMe
 import visibilityModifierDefaultConfig from "../../../shared-unpackaged/visibility-modifier.js"; // eslint-disable-line import/no-restricted-paths
 import RefTracker from "../util/ref-tracker.js";
 import type {Placement} from "../util/types.js";
-import type {TooltipBubbleProps} from "./tooltip-bubble.js";
+import type {PopperElementProps} from "./tooltip-bubble.js";
 
 type Props = {|
     /**
      * This uses the children-as-a-function approach, mirroring react-popper's
-     * implementation, except we enforce the return type to be our
-     * `TooltipBubble` component.
+     * implementation.
      */
-    children: (TooltipBubbleProps) => React.Element<typeof TooltipBubble>,
```

Comment:
I removed the `typeof TooltipBubble` validation as we need to make it work for `PopoverDialog` as well. But I didn't want to import popover files here, so that's way I used `any` instead. Is there a better way to approach this?

Reply:
Templatize the props and pass the element type in?